### ISSUE TITLE
Missing package: tensorflow

### ISSUE THREAD

**Issue by mstfldmr** [piwheels/packages#162]
Package name: tensorflow
Issue type: Missing package
Link to PyPI page: pypi.org/project/tensorflow
Link to piwheels page:  	piwheels.org/project/tensorflow/json
Version: e.g. 2.3.1 
Python version: 3.7
I am the maintainer: No
More information:

**Comment by bennuttall** [comment]
There's no source provided on PyPI, so there's nothing to build from.

We used to get wheels from the tensorflow team but they stopped sending them. Will look into it.

**Comment by KopfKrieg** [comment]
Any updates on this? Or is there something I could do to help? I really, really want to use tensorflow 2.x on my Raspberry Pi/Revolution Pi.

**Comment by mstfldmr** [comment]
@bennuttall the PyPI page has a link to the official repository of https://github.com/tensorflow/tensorflow/tags

Can you add this to your builds?

**Comment by mstfldmr** [comment]
> Any updates on this? Or is there something I could do to help? I really, really want to use tensorflow 2.x on my Raspberry Pi/Revolution Pi.

meanwhile you can cross-compile for Raspi: https://www.tensorflow.org/install/source_rpi#python-3.7

**Comment by KopfKrieg** [comment]
> https://www.tensorflow.org/install/source_rpi#python-3.7

I tried compiling the package a while ago, but it failed with various error messages. Also, which might be a problem, I need a Python 3.5 compatible package, not sure if this will possible after January 2021.

**Comment by mstfldmr** [comment]
@KopfKrieg did you build with following the link I shared? On which kind of a machine did you build?

**Comment by KopfKrieg** [comment]
> @KopfKrieg did you build with following the link I shared? On which kind of a machine did you build?

Yes, I built the package using the link you shared. I've used my homeserver to build it, with an AMD A10-6790K (4x4.0GHz) running Ubuntu 20.04.1 LTS. Not the newest technology, but also not really outdated. Not sure how the author of the instructions built the package in just 30 minutes.

Having a pip package available would still be better, though.

**Comment by mstfldmr** [comment]
It took me 25min today, but on a 96-core EC2 instance :(
It's failing on my Macbook.
